### PR TITLE
Initial setup and configuration for GA Integration

### LINKED ISSUE #44: Initial setup and configuration for GA integration
### Descriptive summary

We need an evaluation of the [built in statistics features](https://github.com/projecthydra/sufia/wiki/Feature-matrix) in Hyrax (below) and a gap analysis performed to get a sense of the amount of work we'll need to port out statistics requirements to our new system.

* Capture usage statistics -- [requires Google Analytics ID to be specified in Sufia initializer](https://github.com/projecthydra/sufia/wiki/Sufia-Management-Guide#capturing-usage)
* Display usage statistics in the UI -- [requires configuration](https://github.com/projecthydra/sufia/wiki/Sufia-Management-Guide#displaying-usage-in-the-ui) and will likely have to change any instance of `sufia` to `hyrax`

### Rationale

We will be porting/re-creating statistics needed for DLDP and RDCP, as well as analytics needed to make data driven decisions going forward. 

### Expected behavior

Out of the box Hyrax GA support. Nothing custom at this time.

### Related work

#41

### PR DESCRIPTION
**Pull request by VivianChu**
Fixes #44  ; refs #44 

Changes proposed in this pull request:
* Update configuration files for google analytics.

@ucsdlib/developers - please review

### CODE REVIEW

File: config/initializers/hyrax.rb [hunk 1]
```diff
@@ -16,13 +16,13 @@
   # Enable displaying usage statistics in the UI
   # Defaults to FALSE
   # Requires a Google Analytics id and OAuth2 keyfile.  See README for more info
-  # config.analytics = false
+  config.analytics = true
```
Comment (mcritchlow) [inline config/initializers/hyrax.rb]: Is this something that should be tweaked on a per-environment basis?

Comment (VivianChu) [reply config/initializers/hyrax.rb]: Yes, I can change it to use the environment variable for the value.

Comment (VivianChu) [reply config/initializers/hyrax.rb]: @mcritchlow - It has been changed to use env variable.

File: config/initializers/hyrax.rb [hunk 2]
```diff
@@ -16,13 +16,13 @@
...
-  # config.analytics = false
+  config.analytics = ENV.fetch('APPS_HORTON_ANALYTICS'){'false'}
 
   # Specify a Google Analytics tracking ID to gather usage statistics
-  # config.google_analytics_id = 'UA-99999999-1'
+  config.google_analytics_id = ENV.fetch('APPS_HORTON_GOOGLE_ID'){'UA-99999999-1'}
 
   # Specify a date you wish to start collecting Google Analytic statistics for.
-  # config.analytic_start_date = DateTime.new(2014,9,10)
+  config.analytic_start_date = DateTime.new(2014,9,10)
```
Comment (mcritchlow) [inline config/initializers/hyrax.rb:25]: I'm curious about what this means exactly. We might want to (in a separate ticket is fine) make a more informed decision about what date to plug in here.

Comment (VivianChu) [reply config/initializers/hyrax.rb:25]: I don't know what date to use either.  So I just use the default date they put in there.

**Comment by mcritchlow** [comment]
@ucsdlib/developers any outstanding concerns here?

**Comment by hweng** [comment]
:+1:

**Comment by lsitu** [comment]
@mcritchlow I am fine with making the decision for the date later. Would you like me you merge it now?

**Comment by mcritchlow** [comment]
@lsitu I just wanted us to not let this PR drift for too long. I agree we need to circle back to the date. Perhaps I'll setup a ticket for that, and unless anyone has any further concerns we could get this PR merged.

**Review by lsitu — APPROVED** [review]

**Pull request merged**